xcape/core.py

```diff
@@ -289,7 +289,7 @@ def _calc_srh_gufunc(*args, **kwargs):
 
 # the numpy version of the algorithm
 def _calc_srh_numpy(*args,
-                    depth = 3000, vertical_lev='sigma',output_var='all'):    
+                    depth = 3000, vertical_lev='sigma',output_var='srh'):    
```

Comment:
changed default to 'srh' - the 'all' option doesn't work right now, and I might just delete it.